src/EA.Weee.Web/Areas/Admin/Controllers/Base/ObligationsBaseController.cs

```diff
@@ -1,10 +1,30 @@
 ﻿namespace EA.Weee.Web.Areas.Admin.Controllers.Base
 {
     using EA.Weee.Security;
+    using EA.Weee.Web.Areas.Scheme.Attributes;
     using EA.Weee.Web.Filters;
+    using EA.Weee.Web.Services;
+    using EA.Weee.Web.Services.Caching;
+    using System;
+    using System.Threading.Tasks;
 
     [AuthorizeInternalClaims(Claims.InternalAdmin)]
+    [ValidatePcsObligationsEnabled]
```

Comment:
Can you add a test to the ObligationsBaseControllerTests to ensure it has the ValidatePcsObligationsEnabled attribute